src/HDInsight/HDInsight/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+This release migrates Microsoft.Azure.Graph SDK to MicrsoftGraph SDK, there is not customer impact.
```

Comment:
This migration might have impact on customers because the permission required to call AAD graph and MS graph APIs are different.
```suggestion
This release migrates Microsoft.Azure.Graph SDK to MicrsoftGraph SDK.
```

Reply:
Fixed